manifests/s/Stoozy/MinecraftLauncher/0.1.1/Stoozy.MinecraftLauncher.yaml

```
@@ -0,0 +1,14 @@
+PackageIdentifier: "Stoozy.MinecraftLauncher"
+PackageVersion: "0.1.1"
+PackageLocale: "en-US"
+Publisher: "Stoozy"
+PackageName: "MinecraftLauncher"
```

```suggestion
PackageName: "sml"
```
It registers as sml in the control panel

---

Yea I guess not